service/interpreter/cadence/worker.go

```diff
@@ -37,12 +41,17 @@ func (iw *InterpreterWorker) Close() {
 	iw.worker.Stop()
 }
 
-func (iw *InterpreterWorker) Start() {
+func (iw *InterpreterWorker) Start(startOptions StartOptions) {
```

Comment:
Could we keep the original `Start` function here and have a second `StartWithOptions`  so that the `launchTemporalService` in `iwf.go` does not need to change?  The `Start` function could just set the default StartOptions and call StartWithOptions - I think that's cleaner overall but in particular keeps launchTemporalService clean

Unless we think we have a use case where launchTemporalService would actually be calling start with different options based on runtime configuration values??

Reply:
That makes sense to me. Definitely, we can keep `Start` as it was and add `StartWithOptions`. I like the idea, thanks for this suggestion!